client/components/ReviewRating/ReviewList/ReviewRating.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import ReviewList from './ReviewList';
+import '../../../css/reviewRating.css';
+
+const ReviewRating = ({ id }) => {
+  const [reviews, setReviews] = useState([]);
+  const [reviewShow, setReviewShow] = useState(2);
+  const [isDoneLoading, setIsDoneLoading] = useState(false);
```

Comment:
As a matter of consistency, can we use [isLoading, setIsLoading] and [isError, setIsError] for our loading and error state handling?

Reply:
sure, I will make the changes.